src/meshdb/settings.py

```diff
@@ -61,12 +56,22 @@
     "https://map.grandsvc.mesh",
     "http://forms.grandsvc.mesh",
```

Comment:
Do we need http?

Reply:
For `.mesh`, yes, unfortunately. That's perhaps something to address.